libpod/pod.go

```diff
@@ -314,15 +314,16 @@ func (p *Pod) CgroupPath() (string, error) {
 			return "", errors.Wrapf(err, "could not get infra")
 		}
 		if ctr != nil {
-			ctr.Start(context.Background(), true)
+			// FIXME why should a call like CgroupPath() start the infra this looks wrong!!!
+			if err := ctr.Start(context.Background(), true); err != nil {
+				return "", err
+			}
```

Comment:
Why does CgroupPath() starts the infra? This looks wrong!

@mheon @giuseppe @cdoern PTAL

Reply:
I agree, this might be a remnant. However, if infra has not started does the cgroupPath return an empty string?